File: Ejercicios Corredor/Trabajo Corredor/Texto.js
class Texto {
  nombre = "Kevin";
  apellido = "Andrey";
  ciudad = "Bogotá";
  pais = "Colombia";
  profesion = "Desarrollador";

  concatenar(a, b) { return `${a} ${b}`; }
  contar(texto) { return texto.length; }
  mayuscula(texto) { return texto.toUpperCase(); }
  minuscula(texto) { return texto.toLowerCase(); }
  contiene(texto, palabra) { return texto.includes(palabra); }
}

File: Ejercicios Corredor/Trabajo Corredor/Texto.java
public class Texto {
    String nombre = "Kevin";
    String apellido = "Andrey";
    String ciudad = "Bogotá";
    String pais = "Colombia";
    String profesion = "Desarrollador";

    public String concatenar(String a, String b) { return a + " " + b; }
    public int contar(String texto) { return texto.length(); }
    public String mayuscula(String texto) { return texto.toUpperCase(); }
    public String minuscula(String texto) { return texto.toLowerCase(); }
    public boolean contiene(String texto, String palabra) { return texto.contains(palabra); }
}
